docs/references/api.rst

```diff
@@ -1087,7 +1087,8 @@ This endpoint shows the Pro services that are enabled on the machine.
                - Description
              * - ``name``
                - *str*
-               - Name of the service
+               - | Name of the service.
+                 | Possible values are: ``cc-eal``, ``cis``, ``esm-apps``, ``esm-infra``, ``fips``, ``fips-updates``, ``livepatch``, ``realtime-kernel``, ``ros``, ``ros-updates``.
```

Comment:
I am missing `usg` here. Should both `usg` and `cis` make it to the list? Maybe some `*` with an explanation or reference to the explanation?

Reply:
Good catch. Since there's one release that has both, I'd say both (but like ``usg``/``cis``). It is probably worth including a short note to say which service is available on which version, but an explanation of the two is not for here imo.

Reply:
`usg` won't appear here. I'll add something short to say that "cis" is listed when "usg" is enabled.